extensions/wikia/LanguageWikisIndex/LanguageWikisIndexController.class.php

```diff
@@ -8,11 +8,29 @@ public function __construct() {
 	}
 
 	public function index() {
+		global $wgIncludeClosedWikiHandler;
 
 		$this->specialPage->setHeaders();
 
-		$this->response->setTemplateEngine( \WikiaResponse::TEMPLATE_ENGINE_MUSTACHE );
+		Wikia::addAssetsToOutput( 'language_wikis_index_scss' );
 
+		$this->setVal( 'langWikis', WikiFactory::getLanguageWikis() );
+
+		if ( !empty( $wgIncludeClosedWikiHandler ) ) {
```

Comment:
I think you should use `isEmptyDomainWithLanguageWikis` defined in `LanguageWikisIndexHooks` (and/or move it to another class). It would not depend on `$wgIncludeClosedWikiHandler` which is a config variable used by another extension.
`isEmptyDomainWithLanguageWikis` just check the current city id so it should be quick and reliable.